specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-09-30/openShiftManagedClusters.json

```diff
@@ -0,0 +1,817 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2019-04-30"
```

Comment:
Version does not match folder name or PR title.

Reply:
Thanks. I'll fix this soon